internal/configs/parser_config_dir_test.go

```diff
@@ -340,12 +340,23 @@ func TestIsEmptyDir_noExist(t *testing.T) {
 	}
 }
 
-func TestIsEmptyDir_noConfigs(t *testing.T) {
-	val, err := IsEmptyDir(filepath.Join("testdata", "dir-empty"))
+func TestIsEmptyDir_noConfigsAndTests(t *testing.T) {
+	val, err := IsEmptyDir(filepath.Join("testdata", "dir-empty"), "")
 	if err != nil {
 		t.Fatalf("err: %s", err)
 	}
 	if !val {
 		t.Fatal("should be empty")
 	}
 }
+
+func TestIsEmptyDir_noConfigsButHasTests(t *testing.T) {
+	// The top directory has no configs, but it contains test files
+	val, err := IsEmptyDir(filepath.Join("testdata", "only-test-files"), "tests")
```

Comment:
👋🏻 I took a look at this PR as I'm currently looking at `init` and was interested. 

I think currently this test is checking for whether files exist in either of `testdata/only-test-files` and `testdata/only-test-files/tests`, but that tests folder doesn't exist. Should there be two versions of this test, with one containing tests in a nested tests folder, similar to `top-dir-only-test-files` versus `top-dir-only-test-files-2` elsewhere in this PR?

Reply:
Yes, Thanks for the callout, we could add a similar test. Done in a0ad5dcc6e. Also renamed them a bit better.

Reply:
Thanks for the rename! When I first looked I found that it wasn't immediately obvious how they differed, and the new names will help avoid that